ios/sdk/src/ExternalAPI.m

```diff
@@ -15,23 +15,55 @@
  */
 
 #import <React/RCTBridgeModule.h>
+#import <React/RCTEventEmitter.h>
 
 #import "JitsiMeetView+Private.h"
 
-@interface ExternalAPI : NSObject<RCTBridgeModule>
+// Events
+static NSString * const hangUpEvent = @"org.jitsi.meet.HANG_UP";
+static NSString * const setAudioMutedEvent = @"org.jitsi.meet.SET_AUDIO_MUTED";
+
+
+@interface ExternalAPI : RCTEventEmitter<RCTBridgeModule>
+@property (class, nonatomic, assign) ExternalAPI *instance;
 @end
 
+static ExternalAPI * _instance;
+
 @implementation ExternalAPI
 
 RCT_EXPORT_MODULE();
 
+- (instancetype)init {
+    self = [super init];
+    _instance = self;
+    
+    return self;
+}
+
++ (ExternalAPI*)instance {
```

Comment:
Try to use `bridge.moduleForName @"ExternalAPI"` instead, in order to get a reference to the current module instance.